pkgs/development/libraries/aspell/default.nix

```diff
@@ -2,6 +2,17 @@
 , searchNixProfiles ? true
 }:
 
+let
+
+  # Source for u-deva.cmap and u-deva.cset: use the Marathi
+  # dictionary like Debian does.
+  devaMapsSource = fetchTarball {
```

Comment:
fetchurl

Reply:
`fetchurl` gives me the plain tarball file but in this case I just need two files from the extracted tarball.

I could use `fetchurl` and extract the files in manually `postInstall`, though, if that’s preferrable?

Reply:
Ah, I see. No, this is fine.

Reply:
Actually, I am pretty sure this is going to fail on Hydra because it runs in restricted-eval mode. Therefore, considering you wanted to have it unpacked, you could use `fetchzip`. It doesn't matter if the archive is not a zip file.

Reply:
Ok, thanks, I’ll try it out.

Reply:
Changed.